File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/TokenApiResponse.java
package com.onscreensync.tvapp.apiresponses;

import com.google.gson.annotations.SerializedName;

public class TokenApiResponse {
    @SerializedName("accessToken")
    private String accessToken;

    @SerializedName("expiresIn")
    private long expiresIn;

    @SerializedName("scope")
    private String scope;

    @SerializedName("tokenType")
    private String tokenType;

    @SerializedName("refreshToken")
    private String refreshToken;

    public String getAccessToken() {
        return accessToken;
    }

    public long getExpiresIn() {
        return expiresIn;
    }

    public String getScope() {
        return scope;
    }

    public String getTokenType() {
        return tokenType;
    }

    public String getRefreshToken() {
        return refreshToken;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/TokenApiResponse.kt
package com.onscreensync.tvapp.apiresponses

import com.google.gson.annotations.SerializedName

data class TokenApiResponse(
    @SerializedName("accessToken")
    val accessToken: String?,
    @SerializedName("expiresIn")
    val expiresIn: Long,
    @SerializedName("scope")
    var scope: String?,
    @SerializedName("tokenType")
    val tokenType: String?,
    @SerializedName("refreshToken")
    val refreshToken: String?
)
